tests/ui/impl-trait/in-trait/issue-102140.next.stderr

```diff
@@ -6,11 +6,7 @@ LL |         MyTrait::foo(&self)
    |         |
    |         required by a bound introduced by this call
    |
-help: consider removing the leading `&`-reference
-   |
-LL -         MyTrait::foo(&self)
-LL +         MyTrait::foo(self)
-   |
```

Comment:
I don't know why exactly this suggestion changed, but if you look below, the new "help: the trait `MyTrait` is implemented for `Outer`" is actually emitted instead, so I don't really consider this to be a regression.

Also this suggestion isn't even right, I think?

Reply:
Yes and even removing `&` doesn't help in any way.